app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt

```diff
@@ -53,7 +53,11 @@ class SyncIndicatorView(context: Context, attrs: AttributeSet) : LinearLayout(co
       }
       is Synced -> {
         val durationToMinsAgo = syncState.durationSince.toMinutes().toInt()
-        syncStatusTextView.text = context.resources.getQuantityString(R.plurals.sync_indicator_status_synced, durationToMinsAgo, durationToMinsAgo)
+        if (durationToMinsAgo == 0) {
```

Comment:
We could convert this block to:
```kotlin
syncStatusTextView.text = if (durationToMinsAgo == 0) {
  context.getString(R.string.sync_indicator_status_synced_just_now)
} else {
  context.getString(R.string.sync_indicator_status_synced, durationToMinsAgo)
}
```

This will ensure that `syncStatusTextView.text` is always set and not forgotten to assign a value if a new conditional branch gets added.

Reply:
Sure.